js/core/structure.js

```diff
@@ -70,25 +70,25 @@ define(
 
         var $a = $("<a/>").attr({ href: "#" + id, "class": "tocxref" })
           .append(isIntro ? "" : $span.clone())
-          .append($kidsHolder.contents()),
-          $item = $("<li class='tocline'/>").append($a);
-        if (conf.maxTocLevel === 0 || level <= conf.maxTocLevel) $ul.append($item);
+          .append($kidsHolder.contents());
+        var $item = $("<li class='tocline'/>").append($a);
+        if (conf.maxTocLevel === 0 || level <= conf.maxTocLevel) $ol.append($item);
         current.push(0);
         var $sub = makeTOCAtLevel($sec, doc, current, level + 1, conf);
         if ($sub) {
           $item.append($sub);
         }
         current.pop();
       }
-      return $ul;
+      return $ol;
     }
 
     return {
       run: function(conf, doc, cb) {
-        if (!conf.tocIntroductory) {
+        if ("tocIntroductory" in conf === false) {
```

Comment:
I dont mind, but I am curious why you prefer this idiom?  It seems inconsistent with the rest of the code.